http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala

```diff
@@ -662,6 +676,77 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
       }
   } yield ()
 
+  private def getLandingZoneResources(billingProfileId: String, userToken: Authorization)(implicit
+    ev: Ask[F, AppContext]
+  ): F[LandingZoneResources] =
+    for {
+      // Step 1: call LZ for LZ id
+      landingZoneOpt <- wsmDao.getLandingZone(billingProfileId, userToken)
+      landingZone <- F.fromOption(
+        landingZoneOpt,
+        AppCreationException(s"Landing zone not found for billing profile ${billingProfileId}")
+      )
+      landingZoneId = landingZone.landingZoneId
+
+      // Step 2: call LZ for LZ resources
+      lzResourcesByPurpose <- wsmDao.listLandingZoneResourcesByType(landingZoneId, userToken)
+
+      aksClusterName <- getLandingZoneResourceName(lzResourcesByPurpose,
```

Comment:
We probably want to read `resourceId` for shared resources. Here's the AKS cluster:
```
        {
          "resourceId": "/subscriptions/3efc5bdf-be0e-44e7-b1d7-c08931e3c16c/resourceGroups/mrg-rtnov14-20221114121959/providers/microsoft.containerservice/managedclusters/lz454127da142e705afb28ada",
          "resourceType": "microsoft.containerservice/managedclusters",
          "region": "eastus2"
        },
```

Unfortunately Back Leo is expecting just the name for all these things, so maybe we split on `/` and take the leaf.